src/Common/src/System/Net/Security/NegotiateStreamPal.Unix.cs

```diff
@@ -413,9 +429,22 @@ Interop.NetSecurityNative.Status status
                 contextFlags = ContextFlagsAdapterPal.GetContextFlagsPalFromInterop(
                     (Interop.NetSecurityNative.GssFlags)outputFlags, isServer: true);
 
-                SecurityStatusPalErrorCode errorCode = done ?
-                    (negoContext.IsNtlmUsed && resultBlob.Length > 0 ? SecurityStatusPalErrorCode.OK : SecurityStatusPalErrorCode.CompleteNeeded) :
-                    SecurityStatusPalErrorCode.ContinueNeeded;
+                SecurityStatusPalErrorCode errorCode;
+                if (done)
+                {
+                    if (NetEventSource.IsEnabled)
+                    {
+                        string protocol = isNtlmUsed ? "SPNEGO-NTLM" : "SPNEGO-Kerberos";
+                        NetEventSource.Info(null, $"AcceptSecurityContext: actual protocol = {protocol}");
```

Comment:
should we pass contextHandle as first parameter to NetEventSource() ? 
e.g. thisOrContext https://github.com/dotnet/corefx/blob/master/src/Common/src/System/Net/Logging/NetEventSource.Common.cs#L207

Reply:
Good point. I copied the pattern from another place in the file. But all those places should use something instead of null. I'll fix up the entire file.